common.py

```diff
@@ -0,0 +1,110 @@
+import csv
+import os
+import urllib.request
+
+from IPython import embed
```

Comment:
is IPython in requirements.txt(will check)

Reply:
It's in the requirements but the IPython import shouldn't be there and I removed it in a later commit.

Reply:
Thanks!